modules/fluid_properties/test/tests/auxkernels/tests

```diff
@@ -17,4 +17,13 @@
     exodiff = 'stagnation_temperature_aux_out.e'
     threading = '!pthreads'
   [../]
+  [./fluid_density_aux]
+    type = 'Exodiff'
+    input = 'fluid_density_aux.i'
+    exodiff = 'fluid_density_aux_out.e'
+    threading = '!pthreads'
```

Comment:
```suggestion
```

You shouldn't need this now, right?

Reply:
I don't actually know why that's there, I just copied the other tests in that file - does the fluid property app only work for certain threading options? But if I can remove it, I'll do that!

Reply:
Whoops, sorry I missed that - had it on my to-do list and it got buried. Should be good now

Reply:
Now that I look at it, all of the similar tests above have threading disabled. I suspect this won't work but we'll see 😆